docs/content/preview/yugabyte-platform/administer-yugabyte-platform/high-availability.md

```diff
@@ -72,7 +72,7 @@ You can configure the active instance as follows:
 
     - If you installed YBA using [Replicated](../../install-yugabyte-platform/install-software/replicated/), locate the CA certificate from the path `/var/lib/replicated/secrets/ca.crt` on the YBA instance.
 
-1. Add the active instance CA certificate to the [YugabyteDB Anywhere trust store](../../security/enable-encryption-in-transit/#add-certificates-to-your-trust-store) of the active instance.
+1. Add the active instance CA certificate to the [YugabyteDB Anywhere trust store](../../security/enable-encryption-in-transit/#add-certificates-to-your-trust-store) of the standby instance.
```

Comment:
Unrelated - are you sure? If true, then do it like the following section, eg

1. Switch to the standby instance.
1. Add the active instance root certificate to the [YugabyteDB Anywhere trust store](../../security/enable-encryption-in-transit/#add-certificates-to-your-trust-store) on the standby instance. This allows the standby to connect to the active instance.

Reply:
I will verify this and create a different PR if needed. @ddhodge . For now I have reverted the change. So This PR has only the EA badge changes.